README.md

```diff
@@ -30,16 +37,22 @@ $ dbt seed
 
 ### Running Instructions:
 
-Run fal flow
+1. Run `dbt run`
 
 ```bash
-$ fal flow run --experimental-models
-## runs the whole graph
+$ dbt run
+## Runs the whole graph including the Python models
 ```
 
-Alternatively run fal flow with [graph selectors](https://docs.getdbt.com/reference/node-selection/graph-operators)
+2. Run `fal flow` to execute the full graph including Python scripts. You can use the dbt [graph selectors](https://docs.getdbt.com/reference/node-selection/graph-operators) and [much more](https://docs.fal.ai/). With `fal flow`, you will not have to run `dbt run` since fal handles the full execution.
```

Comment:
```suggestion
2. Run `fal flow run` to execute the full graph including Python scripts. You can use the dbt [graph selectors](https://docs.getdbt.com/reference/node-selection/graph-operators) and [much more](https://docs.fal.ai/). With `fal flow run`, you will not have to run `dbt run` since fal handles the full execution.
```